virttest/test_setup.py

```diff
@@ -2034,9 +2037,12 @@ def cleanup(self):
             if os.path.exists(self.libvirtd_backup_path):
                 os.rename(self.libvirtd_backup_path, self.libvirtd_path)
             if self.user.count('EXAMPLE'):
-                logging.debug("Delete the created user 'testacl'.")
```

Comment:
@hsmj1412 so you face issue during clean up ? if so would this change could help instead of introducing `add_polkit_user` and without params?
```python
if self.user and not self.user.count('EXAMPLE'):
    logging.debug("Delete the created user 'testacl'.")
    cmd = "userdel -r testacl"
```

Reply:
@balamuruhans no, the issue is when the cases give the 'testacl' instead EXAMPLE, but the testacl is not a existed user.